circle.yml

```diff
@@ -10,3 +10,4 @@ machine:
 test:
   override:
     - ./build/docker/valgrind-clang
+    - if test -f ./test-suite.log; then cat ./test-suite.log; fi
```

Comment:
Strictly speaking this is not part of the pull request but was helpful to debug it.